src/SelectField/SelectField.js

```diff
@@ -180,6 +185,7 @@ class SelectField extends Component {
       <TextField
         {...other}
         style={style}
+        defaultText={hintText || floatingLabelText ? '' : defaultText}
```

Comment:
Snap, you're right. It was intended to be [passed for `DropDownMenu`](https://github.com/callemall/material-ui/pull/4902/commits/86902ab205600ae8c81f4b3981131039f6cc2126#diff-b2076f9d48e2e38b6e48a6226571b12dR206).

Does this parameter still make sense or we'll use the first option's value?